README.md

```diff
@@ -42,9 +43,36 @@ or using CakePHP's console:
 ./bin/cake plugin load BootstrapUI
 ```
 
-## AppView Setup
-Then for a complete setup, just make your `AppView` class extends `BootstrapUI\View\UIView`.
-The `src\View\AppView.php` then will be as the following: 
+> The BootstrapUI plugin does not come bundled with Twitter Bootstrap. Instructions for installing the framework are below.
+
+## Running Tests
+
+You can run the tests for BootstrapUI by doing the following:
+
+```
+    $ composer install
+    $ ./vendor/bin/phpunit
+```
+
+## Some Handy Links
```

Comment:
I am referring to the whole "Some Handy Links" section. I don't think it's necessary.